doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -182,8 +182,31 @@ To bring a stalled Target node to idle state, use the ``mesh models dfu cli canc
 
 .. _bluetooth_mesh_dfu_eval_md:
 
-Composing the firmware metadata
-*******************************
+Generating the firmware metadata
+********************************
+
+Automated metadata generation
+=============================
+
+The most efficient way to generate the required DFU metadata is to use the DFU metadata extraction script integrated in the NCS build system. By enabling the ``CONFIG_BT_MESH_DFU_METADATA_ON_BUILD`` option in the
+application, the metadata will be automatically parsed from the .elf and .config file. The parsed data is put into a DFU zip-archive in the build folder of the application in a JSON format. Additionally, the
+metadata string, required by the ``mesh models dfu metadata encode`` command, will be printed in the command line window when the application is built.
```

Comment:
> required by the ``mesh models dfu metadata encode``

It is required by `dfu slot add`:
https://github.com/nrfconnect/sdk-nrf/blob/b962460658b241014e4c6274e03cb52f2b1d5ed6/doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst?plain=1#L52-L58

Reply:
Right... That is what I meant to write 👍